examples/digits_dann_lightn/main.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
filename: how about calling it `tutorial.ipynb`. Two `mains` could be confusing to some.

Reply:
Good point. Changed.